### PR TITLE
Revert "Create Managed Network Powershell"

### PR DESCRIPTION
**Pull request by isra-fel**
Reverts Azure/azure-powershell#10195

This is for public preview.
Code is now on branch `Az.ManagedNetwork-Preview`.

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by isra-fel** [comment]
@yanfa317 FYI

File: tools/StaticAnalysis/Exceptions/Az.ManagedNetwork/SignatureIssues.csv [hunk 1]
```diff
@@ -1,2 +0,0 @@
-"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
```
Comment (markcowl) [inline tools/StaticAnalysis/Exceptions/Az.ManagedNetwork/SignatureIssues.csv]: seems liek this file should be completedlt removed.  Also, the 'PS-VSPrompt.lnk file above should not be included in this PR.

Comment (isra-fel) [reply tools/StaticAnalysis/Exceptions/Az.ManagedNetwork/SignatureIssues.csv]: @markcowl
Yes, `SignatureIssues.csv` is indeed deleted.

"PS-VSPrompt.lnk" was deleted by accident in the original PR. This PR will recover it back.
